### PR TITLE
Adding trigrams.py (Exercise 3) and some other Lesson 04 files (mailroom2.py has already been reviewed)

### PR DESCRIPTION
**Pull request by mokiko81**
I have added lesson 04 assignments and exercises. The lesson 04 mailroom2.py is complete but I had some difficulty using formatting with the '.format(**dictionary)'. The file is fully functional but I used line by line formatting to create the .txt files instead of the (**dictionary).

### CODE REVIEW

**Comment by mokiko81** [comment]
I have already had Mailroom2.py reviewed. I am submitting my Trigrams exercise for review:
  Trigrams exercise was incredibly cool, I didn't splice the read file any further because there was some inconsistencies between different ebooks. Slight differences like:
***START OF **THE** PROJECT GUTENBERG EBOOK ... 
*** START OF **THIS** PROJECT GUTENBERG EBOOK ...

**Comment by natasha-aleksandrova** [comment]
@charlesrobison can you please take another look?

File: students/adam_strong/lesson04/trigrams.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# Trigrams - Assingment 3
+import random
+import os
```
Comment (charlesrobison) [inline students/adam_strong/lesson04/trigrams.py:4]: os imported but not used

File: students/adam_strong/lesson04/trigrams.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    try:
+        filename = sys.argv[1]
+    except IndexError:
+        print("You must pass in a filename")
+        sys.exit(1)
+    words = get_file()
+    words = clean_words(words)
+    trigrams = build_trigrams(words)
+    output = rand_story(trigrams)
+    print(output)
```
Comment (charlesrobison) [inline students/adam_strong/lesson04/trigrams.py:85]: Works well!

**Pull request merged**